iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -248,17 +250,30 @@ void updateUser(EventProcessingRequest request) throws IOException {
         if (attributes == null) {
             return null;
         }
+        Map<String, String> attributesWithReserved = convertReservedAttributes(attributes);
         
         if (coerceStringsToScalars) {
-            return attemptTypeConversion(attributes);
+            return attemptTypeConversion(attributesWithReserved);
         } else {
             Map<String, Object> mapObj = new HashMap<String, Object>();
-            mapObj.putAll(attributes);
+            mapObj.putAll(attributesWithReserved);
 
             return mapObj;
         }
     }
 
+    private static Map<String, String> convertReservedAttributes(Map<String, String> attributes) {
+        Map<String, String> convertedAttrs = new HashMap<String, String>();
+        convertedAttrs.putAll(attributes);
+        if (convertedAttrs.containsKey(MPARTICLE_RESERVED_PHONE_ATTR)) {
+            String phoneNumber = convertedAttrs.get(MPARTICLE_RESERVED_PHONE_ATTR);
+            String formattedNumber = phoneNumber.replaceAll("[\\D]", "");
```

Comment:
Looks like we do require `+` prefix for international numbers - maybe we should keep the `+` sign?

Reply:
https://support.iterable.com/hc/en-us/articles/211970843-SMS-overview#user-fields-required

Reply:
Thanks for catching that. I updated the formatting to allow the "+" character.